### DISCUSSION TITLE
Add ability to text to voice answer when send voice message

### DISCUSSION THREAD

**Discussion by khanfar** [n3d1117/chatgpt-telegram-bot#206] [Ideas]
Hi , I'm thinking when we send voice message and the bot reply with text and voice too . I see this example bot hope it help for this idea : @AI_Text_to_Voice_Speech_Bot

**Comment by n3d1117** [comment]
Will definitely explore this option, maybe using https://play.ht

**Comment by JozhB** [comment]
Would be great. To add to the idea, it might be possible to explore options with Google Text-To-Speech or Azure API, don't you think?
